opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskTrackingPopover.js

```diff
@@ -251,6 +251,7 @@ class RiskTrackingPopover extends Component {
       R.pathOr([], ['logged_by']),
       R.mergeAll,
     )(node);
+    console.log('Node', node)
```

Comment:
Is this needed?

Reply:
That should go away but I think we will need a ticket later to go clean up all of the console.logs

Reply:
I'm ok with letting this go through IF a new ticket to clean up is created